test/distributed/_composable/test_checkpoint.py

```diff
@@ -11,8 +11,6 @@
 from torch.distributed._composable import checkpoint
```

Comment:
Remove the `use_reentrant` kwarg in `test_checkpoint_fsdp_submodules_non_reentrant()`:
```
test_model.u1.seq = checkpoint(test_model.u1.seq, use_reentrant=use_reentrant)
```